### PR TITLE
remove unused access report models

### PR DESCRIPTION
**Pull request by davidsbailey**
Finishes https://codedotorg.atlassian.net/browse/ACQ-380.  As a result of this, we will no longer import state cs offering data when we import data from CSV into the schools table.

## Testing story
* existing test coverage in school_test.rb
* code searches to verify there are no remaining references to the removed models

### CODE REVIEW

File: dashboard/test/models/school_test.rb [hunk 1]
```diff
@@ -295,12 +283,8 @@ class SchoolTest < ActiveSupport::TestCase
   def clear_schools_and_dependent_models
     # Clear tables with hard dependencies (ie, MySQL foreign keys)
     # on the schools table.
-    Census::ApSchoolCode.delete_all
-    Census::IbSchoolCode.delete_all
     Census::CensusOverride.delete_all
```
Comment (megcrenshaw) [inline dashboard/test/models/school_test.rb]: I'm seeing in the engineering plan that the `census_overrides` table will be removed, but I don't see mention of the model. Can this be deleted as well?

**Review by megcrenshaw — APPROVED** [review]
Wahooo! Looks good. Just one question:
I'm seeing some mentions of these tables (like `ap_cs_offerings`) [in the README](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/census/README.md?plain=1#L20). Is that tracked?

File: dashboard/test/models/school_test.rb (hunk 1 above)
Comment (davidsbailey) [reply dashboard/test/models/school_test.rb]: great catch! removed

**Comment by davidsbailey** [comment]
> Wahooo! Looks good. Just one question: I'm seeing some mentions of these tables (like `ap_cs_offerings`) [in the README](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/census/README.md?plain=1#L20). Is that tracked?

yep! that's fixed in https://github.com/code-dot-org/code-dot-org/pull/50561

File: dashboard/test/models/school_test.rb (hunk 1 above)
Comment (megcrenshaw) [reply dashboard/test/models/school_test.rb]: Great! Changes looks good 👍

**Pull request merged**